src/api/config-manager.js

```diff
@@ -12,9 +12,11 @@ const getContract = () => {
 };
 
 module.exports = {
+  getContract,
+
   async bodhiTokenAddress() {
     try {
-      return getContract().methods.bodhiTokenAddress().call();
+      return this.getContract().methods.bodhiTokenAddress().call();
```

Comment:
do you need to add `this.` to all of them? dont they reference the same function if you dont?

Reply:
Looks like if without `this.`, the test won't catch the stub version.